mythril/analysis/modules/transaction_order_dependence.py

```diff
@@ -1,5 +1,6 @@
 import logging
 import re
+import copy
```

Comment:
Is this necessary?

Reply:
Yes, copy.deepcopy() is used in _can_change() in this file.